### PR TITLE
fixes missing async polyfill for the browser

### PR DESCRIPTION
**Pull request by argyleink**

### CODE REVIEW

**Comment by leplatrem** [comment]
Oh that's a dupe of #550 :)

File: package.json [hunk 1]
```diff
@@ -128,7 +128,9 @@
     "babel-loader": "^6.2.1",
     "babel-plugin-transform-es2015-modules-commonjs": "^6.4.3",
     "babel-plugin-transform-object-rest-spread": "^6.3.13",
+    "babel-plugin-transform-runtime": "^6.15.0",
     "babel-preset-es2015": "^6.3.13",
+    "babel-runtime": "^6.11.6",
```
Comment (leplatrem) [inline package.json]: isn't babel-runtime a dep of the plugin?

Comment (leplatrem) [inline package.json]: yes it is ;)
https://github.com/babel/babel/blob/master/packages/babel-plugin-transform-runtime/package.json#L12

**Comment by argyleink** [comment]
nice! should have searched your open pr's first

**Pull request closed without merging**